tools/noise/subr.sh

```diff
@@ -446,11 +446,12 @@ export_large_jpeg() {
 	input=$1
 	output=$2
 	xmp="$input.xmp"
-
+        xmp_profiling="$scriptdir/profiling-shot.xmp"
+	
 	tool_installed darktable-cli
 
 	rm -f "$output" "$xmp"
-	darktable-cli "$input" "$output" &> /tmp/dt_output.log  || echo $(cat /tmp/dt_output.log); exit 1
+	darktable-cli "$input" "$xmp_profiling" "$output" --core --conf plugins/lighttable/export/iccprofile=image --conf plugins/lighttable/export/style=none
```

Comment:
this is great, thanks for doing it, I had it on my agenda for a while, but didn't find the time yet...

I commented on your earlier PR about the rational for the conditional echo + exit statement:
https://github.com/darktable-org/darktable/pull/11468#discussion_r851646577

It's fine if you remove it in your PR, as it has been poorly implemented to be honest, I'll find a better way how to achieve this functionality.

Reply:
Turns out running a darktable-cli instance while the darktable GUI application using the local library is possible by this extra cli argument: `--apply-custom-presets false` (see https://github.com/darktable-org/darktable/issues/2890#issuecomment-657190553).
This will also remove the necessity for aborting the process as I can't think of any other reason why darktable would not be able to export the JPG image.
```suggestion
	darktable-cli "$input" "$xmp_profiling" "$output" --core --conf plugins/lighttable/export/iccprofile=image --conf plugins/lighttable/export/style=none --apply-custom-presets false
```

Reply:
Would be great if you could also update the other invocation of darktable-cli here:
https://github.com/darktable-org/darktable/blob/5a36c7ae67cfdff8c87527c10116d34c28f82370/tools/noise/darktable-gen-noiseprofile#L149

Reply:
Both invocations updated with `--apply-custom-presets false`.